### PR TITLE
support where clause in imagetable.show() and fix a bug to show()

### PR DESCRIPTION
**Pull request by wenyushi**
The changes will offer a flexible way to choose the images to be shown. Users write SAS where clause to make it. One example is dlpy/tests/test_imagetable.py test_show(). Besides, the push will fix a bug when nimages is greater than the number of observations.

### CODE REVIEW

File: dlpy/images.py [hunk 1]
```diff
@@ -339,12 +339,19 @@ def show(self, nimages=5, ncol=8, randomize=False, figsize=None):
             columns in the plots.
         randomize : bool, optional
             Specifies whether to randomly choose the images for display.
-        figsize: int, optional
+        figsize : int, optional
             Specifies the size of the fig that contains the image.
+        where : string, optional
```
Comment (ethem-kinginthenorth) [inline dlpy/images.py]: we need an example here

File: dlpy/tests/test_imagetable.py [hunk 2]
```diff
@@ -206,3 +204,12 @@ def test_as_random_patches(self):
         self.assertTrue(int(out[4]) == 200)
         self.assertTrue(int(out[5]) == 200)
         self.assertTrue(int(out[6]) == 200)
+
+    def test_show(self):
```
Comment (ethem-kinginthenorth) [inline dlpy/tests/test_imagetable.py]: an example where where clause fails. we might need to catch that error. thanks

Comment (wenyushi) [reply dlpy/tests/test_imagetable.py]: Yes. We can either give a warning message and ignore where clause or response an error. Which is better?

Comment (ethem-kinginthenorth) [reply dlpy/tests/test_imagetable.py]: I think this one is better: give a warning message and ignore where clause

**Pull request merged**